### PR TITLE
Convert feedback cukes to rspec

### PR DESCRIPTION
**Pull request by alanth**

### CODE REVIEW

**Comment by alanth** [comment]
I wanted to use Site.feedback_email in the hardcoded email addresses, but couldn't get it to work and ran out of time

File: spec/features/feedback_spec.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'Feedback', type: :feature do
+  before(:each) { visit feedback_url }
+  
+  it 'Submit feedback' do
+    @feedback = Feedback.new(:name => "Joe Public", :email => "foo@example.com",
+      :comment => "I can't submit a petition for some reason", :petition_link_or_title => 'link')
```
Comment (jnormington) [inline spec/features/feedback_spec.rb]: This should and can be moved into a let like so `let(:feedback)  { Feedback.new(name: 'Joe Public', email: 'foo@example.com', comment: 'I can't submit a petition for some reason', petition_link_or_title: 'link') }`. Then all references to `@feedback` will just be `feedback...`

File: spec/features/feedback_spec.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    fill_in "feedback[email]", :with => @feedback.email
+    fill_in "feedback[petition_link_or_title]", :with => @feedback.petition_link_or_title
+    fill_in "feedback[comment]", :with => @feedback.comment
+
```
Comment (jnormington) [inline spec/features/feedback_spec.rb:12]: With regards to the fill_in can we use the relevant label name for example `fill_in 'Comments'` to make this a little more readable where possible.

File: spec/features/feedback_spec.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    expect(page).to have_content("Thank")
+    
+    expect(unread_emails_for('petitionscommittee@parliament.uk').size).to eq parse_email_count(1)
+    open_email('petitionscommittee@parliament.uk')
+    expect(current_email.default_part_body.to_s).to include(@feedback.petition_link_or_title)
+    expect(current_email.default_part_body.to_s).to include(@feedback.email)
+    expect(current_email.default_part_body.to_s).to include(@feedback.comment)
+  end
+  
+  it 'Check required fields' do
```
Comment (jnormington) [inline spec/features/feedback_spec.rb:23]: The it block description should be lowercase to form a nice sentence from the top level describe block. Also the it description is a little vague on what it is testing.

**Comment by jnormington** [comment]
Left some comments and also not sure you meant to commit .env files and database.yml?

**Pull request closed without merging**